docker/Dockerfile

```diff
@@ -44,6 +43,9 @@ RUN mkdir ${HOME}/.jupyter && \
 FROM base AS cpu
 
 # Setup Conda environment
+RUN apt-get update && \
+    apt-get install -y python-pip && \
+    pip install cmake
 RUN python recommenders/scripts/generate_conda_file.py --name base
```

Comment:
Would it be better to install this via conda at the end so it's available for all images and integrated into the conda environment?